packages/stg-definitions-v2/src/constant.ts

```diff
@@ -709,6 +724,13 @@ export const REWARDS: RewardsConfig = {
             [EndpointId.ZKCONSENSYS_V2_MAINNET]: {
                 address: '0x808d7c71ad2ba3FA531b068a2417C63106BC0949',
             },
+
+            //
+            // Testnet
+            //
+            [EndpointId.MANTLESEP_V2_TESTNET]: {
+                address: 'TODO?',
```

Comment:
Hmm this may not be needed here actually because we will be testing with RewardTokenMock

Reply:
Thanks!